components/collection/CollectionList.vue

```diff
@@ -168,3 +172,9 @@ watch(
   }
 )
 </script>
+
+<style lang="scss" scoped>
+.skeleton-fixed-width {
+  width: 80px;
```

Comment:
There is already a `width` prop on `NeoSkeleton`, have you tried something like: 
```ts
<NeoSkeleton v-else-if="isLoading" no-margin width="80px" />
```

Reply:
@roiLeo I tried, it doesn't work. `width` applied to the skeleton item width, but width of skeleton container is still 100% that will cause some odd style issue.
![CleanShot 2023-05-11 at 18 20 51@2x](https://github.com/kodadot/nft-gallery/assets/128157824/9c5de160-2098-4e7e-8c1e-60439c2ea02e)

It is true that there are other ways to solve this problem: for example, making the filter tag on the left side not wrap, and then making the skeleton item on the right side right-aligned. But the easiest way is to give NeoSkeleton a separate container and specify its width.

Reply:
Understandable, maybe rename class to skeleton-container-fixed-width